labelCloud/model/point_cloud.py

```diff
@@ -182,6 +185,35 @@ def from_file(
             write_buffer,
         )
 
+    def validate_segmentation_label(self) -> None:
+        unique_label_ids = set(np.unique(self.labels))  # type: ignore
+        unique_class_ids = set(c.id for c in LabelConfig().classes)
+        if not unique_class_ids.issuperset(unique_label_ids):
+            msg = QMessageBox()
+            msg.setWindowTitle("Invalid segmentation label")
+            msg.setText(
+                f"Segmentation labels {unique_label_ids} of `{self.path}` don't match with the label config {unique_class_ids}."
+            )
+            labels_to_replace = unique_label_ids.difference(unique_class_ids)
+            msg.setInformativeText(
+                f"""
+                Do you want to overwrite 
+                the mismatch labels {labels_to_replace} with 
```

Comment:
Maybe "undefined labels" instead of "mismatch labels"?

Reply:
done here [84558e1](https://github.com/ch-sa/labelCloud/pull/125/commits/84558e1f25f2dc93682e00ae50dae48720426319)